os/arch/arm/src/imxrt/imxrt_hyperflash.c

```diff
@@ -446,6 +446,20 @@ status_t imxrt_flexspi_nor_hyperbus_read(FLEXSPI_Type *base, uint32_t addr, uint
 	return status;
 }
 
+/************************************************************************************
+ * Name: up_dumptoflash
+ *
+ * Description:
+ *   Wrapper function for flexispi write call
+ *
+ ************************************************************************************/
+#if defined(CONFIG_BOARD_CRASHDUMP)
+uint32_t up_dumptoflash(uint32_t addr, uint32_t *buffer, uint32_t bytes)
```

Comment:
Doesn't it need to add this function also in 'imxrt_norflash.c'?

Reply:
NXP did not provide the norflash write api interface for 1020 board. I will be requesting them for the same.